detectors/node/opentelemetry-resource-detector-docker/package.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "@opentelemetry/resource-detector-docker",
+  "version": "1.0.0",
+  "description": "Opentelemetry resource detector to get docker containerId",
```

There may be other things added in the future. I would make this more general
```suggestion
  "description": "OpenTelemetry resource detector for docker resource attributes",
```

---

makes sense - done